File: C/normal loop/reverse.c
#include<stdio.h>
int main()
{
    int n;
    printf("enter the value :");
    scanf("%d",&n);
     int reverse=0;
     int k=n;
     while(n>0){
        int a;
        a=n%10;
        reverse=reverse*10+a;
        n=n/10;
     }
     printf("reverse is :%d",reverse+k);
     return 0;
}

File: JAVASCRIPT/loops.js/reverse.js
const prompt=require('prompt-sync')()

let n = Number(prompt('enter the number -->'))
let reverse=0,m;
for(;n>0;){
    m=n%10;

    n=Math.floor(n/10)
    reverse=(reverse*10)+m;
}
console.log(reverse);
